packages/evolution-generator/src/scripts/generate_choices.py

```diff
@@ -74,9 +78,17 @@ def generate_choices(input_file: str, output_file: str):
                 "spread_choices_name": spread_choices_name,
             }
 
-            # Add the 'conditional' field only if it exists
-            if "conditional":
-                choice["conditional"] = conditional
+            # Add conditional field to choice object if it exists
+            if conditional is not None and conditional.endswith("CustomConditional"):
+                # Check to see if the conditional finish with 'CustomConditional'
+                has_custom_conditionals_import = True
+                choice["conditional"] = f"customConditionals.{conditional}"
+            elif conditional is not None and not conditional.endswith(
```

Comment:
Puisque c'est un elif, le 2e énoncé (not condition.endWith(blabla)) n'est pas nécessaire

Reply:
J'ai changé à if, car je veux éviter qu'il se passe quelque chose si conditional est None. 

```python
# Add conditional field to choice object if it exists
if conditional is not None and conditional.endswith("CustomConditional"):
    # Check to see if the conditional finish with 'CustomConditional'
    has_custom_conditionals_import = True
    choice["conditional"] = f"customConditionals.{conditional}"
elif conditional is not None and not conditional.endswith(
    "CustomConditional"
):
    # Check to see if the conditional does not finish with 'CustomConditional'
    has_conditionals_import = True
    choice["conditional"] = f"conditionals.{conditional}"
```